api/app/signals/apps/search/signal_receivers.py

```diff
@@ -0,0 +1,22 @@
+from django.conf import settings
+from django.dispatch import receiver
+
+from signals.apps.search.tasks import save_to_elastic
+from signals.apps.signals.managers import (
+    create_child,
+    create_initial,
+    update_category_assignment,
+    update_location,
+    update_priority
+)
+
+
+@receiver([create_initial,
+           create_child,
+           update_location,
+           update_category_assignment,
+           update_priority], dispatch_uid='search_add_to_elastic')
+def add_to_elastic_handler(sender, signal_obj, **kwargs):
+    # Add to elastic
+    if settings.FEATURE_FLAGS.get('SEARCH_BUILD_INDEX', False):
```

Comment:
add to index when flag is set to `False`?

Reply:
? No, when it is True we can add it to the index. The default is False.